### ISSUE TITLE
Mounted_Swimming

### ISSUE THREAD

**Issue by delgelato** [vis2k/uMMORPG_CE#11]
In PlayerCharacterControllerMovement.752
```
// move with acceleration (feels better)
horizontalSpeed = AccelerateSpeed(inputDir, horizontalSpeed, swimSpeed, inputDir != Vector2.zero ? swimAcceleration : swimDeceleration);
moveDir.x = desiredDir.x * horizontalSpeed;
moveDir.z = desiredDir.z * horizontalSpeed;

// move
moveDir.x = desiredDir.x * swimSpeed;
moveDir.z = desiredDir.z * swimSpeed;
```

Under // move shouldn't be there? Or it's supposed to be a move without acceleration?
Either way, only 1 of them should be happening here

**Comment by miwarnec** [comment]
nice find, checking

**Comment by miwarnec** [comment]
fixed in V2.15, nice find!
